Jenkinsfile

```diff
@@ -1023,9 +1023,7 @@ pipeline {
                         unstash 'js-exe'
 
                         runShell("""
-                            wget https://github.com/tcnksm/ghr/releases/download/v0.12.1/ghr_v0.12.1_linux_amd64.tar.gz
-                            tar -zxvpf ghr_v0.12.1_linux_amd64.tar.gz && rm bin/ghr_v0.12.1_linux_amd64.tar.gz || true
-                            ./ghr_v0.12.1_linux_amd64/ghr -r stanc3 -u stan-dev -recreate ${tagName()} bin/
+                            gh release create ${tagName()} --latest --target master --generate-notes ./bin/*
```

Comment:
The `release create` command fails if the tag already exists, so I think you need 
```
gh release delete ${tagName()} --cleanup-tag -y || true
```
First.

Also, I don't think `--generate-notes` is necessary/generates output we want. Something like 

`--notes "$(git log --pretty=format:'nightly: %h %s' -n 1)"` seems nicer

Reply:
Ah, I've read this flag:
``` 
--verify-tag 
Abort in case the git tag doesn't already exist in the remote repository
```

I thought it would proceed, misled me a bit.
Agree with the notes.